40/generated/topic_config.html

```diff
@@ -111,7 +111,12 @@ <h4><a id="follower.replication.throttled.replicas"></a><a id="topicconfigs_foll
 </li>
 <li>
 <h4><a id="index.interval.bytes"></a><a id="topicconfigs_index.interval.bytes" href="#topicconfigs_index.interval.bytes">index.interval.bytes</a></h4>
-<p>This setting controls how frequently Kafka adds an index entry to its offset index. The default setting ensures that we index a message roughly every 4096 bytes. More indexing allows reads to jump closer to the exact position in the log but makes the index larger. You probably don't need to change this.</p>
+<p>This setting controls how frequently Kafka adds entries to its offset index and, conditionally, to its time index. After appending a batch of messages, if the total bytes written since the last index entry exceed this value, Kafka will:</p>
```

Comment:
Hi @shmily7829, thanks for the PR! 
Normally we'll fix the config doc in https://github.com/apache/kafka/blob/269e2d898b76b5c9c58232c35bee805ceacc2ead/clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java#L117

This doc you are editing here is generated by script.

Reply:
Hi @brandboat,
Thank you for the reminder.
I will make the corrections by this weekend.

Reply:
The updated PR has been submitted at https://github.com/apache/kafka/pull/19652.
Therefore, I’ll go ahead and close this one.
@brandboat Thank you for your time and review!